pkgs/by-name/ui/uiua/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  pkgs,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "uiua";
+  version = "unstable-2023-09-27";
+
+  src = fetchFromGitHub {
+    owner = "uiua-lang";
+    repo = "uiua";
+    rev = "f5cd0bc4f1f2e34c8b75324c23e9b2db3f1c6727";
+    hash = "sha256-Os9soRokxxlh/+Jd5L0K/NWxiSBc8ILe6MAA92W95LI=";
+  };
+
+  # https://github.com/uiua-lang/uiua/issues/8
+  cargoLock.lockFile = ./Cargo.lock;
```

Comment:
A `Cargo.lock` file was added in a recent commit, so I would recommend updating.

Reply:
Ohh, well, here I go, one moment.

Reply:
Alright, it should be removed now, and it did seem to still build fine.